web/sites/default/settings.php

```diff
@@ -37,3 +29,17 @@
 
 $config['openkj.settings']['api_key'] = '12345';
 $config['openkj.settings']['default_venue'] = '1';
+
+// configuration split
+if ($settings['server_environment'] === 'live') {
+  $config['config_split.config_split.development']['status'] = FALSE;}
+else { $config['config_split.config_split.development']['status'] = TRUE; }
```

Comment:
The example I gave was not meant to be the actual solution...in order for it to work, `$settings['server_environment']` needs to be set as well. No worries! I am adding that logic at the top of the file (taken from another project I was part of [FHAO] and will make a new commit.